src/classes_def.xml

```diff
@@ -1,21 +1,36 @@
 <lcgdict>
-    <class name="HH::Lepton">
-        <field name="hlt_already_tried_matching" transient="true"/>
+    <class name="HH::Lepton" ClassVersion="10">
+     <version ClassVersion="10" checksum="1833596599"/>
```

Comment:
`scram build updateclassversion` does it automatically for you. If you change the content of the structure, `scram` checks during the build that the class checksum is still the one mentioned in the xml file. If not, the build fails and you have to bump the version by one and recompute the checksum with `scram build updateclassversion`. But what clearly is the checkum I don't know, maybe a decimal representation of a `md5`?